src/benchmarks/gc/src/exec/GCPerfSim/tests_to_be_adapted/cwt/Program.cs

```diff
@@ -75,9 +75,10 @@ static void Main2(string[] args)
                     
                     SomeKey sk = new SomeKey(i.ToString(), objSize);
                     SomeSet ss = new SomeSet(i.ToString(), objSize);
-                    s_cwt.AddOrUpdate(sk, ss);
+                    // s_cwt.AddOrUpdate(sk, ss); // Not available in .net 4.7.2
```

Comment:
Is it OK to not add the values here at all?

Reply:
This is dead code right now -- see the exception below. These tests need to be adapted so the problem could be fixed then -- possibly this could only be available in a coreclr build.